extensions/metadata/interface/src/main/java/org/exist/storage/md/xquery/Check.java

```diff
@@ -55,8 +52,11 @@
 	
 	public final static FunctionSignature signature =
 		new FunctionSignature(
-			new QName("create", MDStorageManager.NAMESPACE_URI, MDStorageManager.PREFIX),
-			"",
+			new QName("check", MetaData.NAMESPACE_URI, MetaData.PREFIX),
```

Comment:
it's possible, but that resource will not be accessible by metadata storage methods and UUID reference.